Rubberduck.Core/UI/Refactorings/MoveToFolder/MoveMultipleToFolderView.xaml.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.MoveToFolder;
+
+namespace Rubberduck.UI.Refactorings.MoveToFolder
+{
+    public partial class MoveMultipleToFolderView : IRefactoringView<MoveMultipleToFolderModel>
+    {
+        public MoveMultipleToFolderView()
+        {
+            InitializeComponent();
+
+            Loaded += (o, e) =>
+                {
+                    MoveToFolderTextBox.Focus();
+                    MoveToFolderTextBox.SelectAll();
+                };
```

Comment:
Worrying about handler holding up GC here too

Reply:
Could we simply unregister the handler in the handler itself?